# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/GCAPI.h
# Commit: f207beed6252
# Full Hash: f207beed6252d85e5a64cc768eaef9d8a47b4912
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-28 04:43:55
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 4: Remove option to set maximum mark stack capacity in release builds r=sfink
#   
#   Currently we have a GC parameter that allows setting a maximum mark stack
#   capacity. This is only ever used by test code, not in the browser. This
#   requires extra unnecessary work in release builds if we move to a different
# ==============================================================================

diff -r 4746d15d7a9f -r f207beed6252 js/public/GCAPI.h
--- a/js/public/GCAPI.h	Thu Oct 27 18:23:03 2022 +0000
+++ b/js/public/GCAPI.h	Thu Oct 27 18:23:03 2022 +0000
@@ -130,14 +130,6 @@
   JSGC_SLICE_TIME_BUDGET_MS = 9,
 
   /**
-   * Maximum size the GC mark stack can grow to.
-   *
-   * Pref: none
-   * Default: MarkStack::DefaultCapacity
-   */
-  JSGC_MARK_STACK_LIMIT = 10,
-
-  /**
    * The "do we collect?" decision depends on various parameters and can be
    * summarised as:
    *